files/zh-cn/learn/javascript/building_blocks/looping_code/index.html

```diff
@@ -312,21 +312,21 @@ <h2 id="使用break退出循环">使用break退出循环</h2>
 
 <p>然后是JavaScript：</p>
 
-<pre class="brush: js">var contacts = ['Chris:2232322', 'Sarah:3453456', 'Bill:7654322', 'Mary:9998769', 'Dianne:9384975'];
-var para = document.querySelector('p');
-var input = document.querySelector('input');
-var btn = document.querySelector('button');
+<pre class="brush: js">const contacts = ['Chris:2232322', 'Sarah:3453456', 'Bill:7654322', 'Mary:9998769', 'Dianne:9384975'];
+const para = document.querySelector('p');
+const input = document.querySelector('input');
+const btn = document.querySelector('button');
 
-btn.addEventListener('click', function(){
-  var searchName = input.value;
+btn.addEventListener('click', function() {
+  let searchName = input.value.toLowerCase();
   input.value = '';
   input.focus();
-  for (var i = 0; i &lt; contacts.length; i++) {
-    var splitContact = contacts[i].split(':');
-    if (splitContact[0] === searchName) {
+  for (let i = 0; i < contacts.length; i++) {
```

Comment:
Please keep the symbol encoded `<` should be `&lt;`, the safer way is just copy and paste from the en content.